manifests/init.pp

```diff
@@ -601,7 +601,7 @@
       # is applied.
       # TODO - make it run only once
       exec { 'pcs_cluster_auth':
-        command => "pcs cluster auth ${node_string} ${auth_credential_string}",
+        command => "pcs host auth ${node_string} ${auth_credential_string}",
```

Comment:
I'm not a corosync expert. Can you explain a bit why do you this change?

Reply:
Syntax has changed from previous version. See 'man pcs' -> Changes in pcs-0.10.

Reply:
but for which operating systems? as far as I know it works fine on the operating systems listed in the metadata.json. If this is version specific we could use a fact for the corosync/pcs version and use the correct command based on that.

Reply:
of course this makes sense. i had to make the change for ubuntu 20.04 because ubuntu 16.04 has no maintenance anymore

Reply:
maybe you can take a look at the work in https://github.com/voxpupuli/puppet-corosync/pull/499 . That already looks promising. It would be nice if we could get this implemented without breaking backwards compatibility.